1.9/administration/securing-your-cluster.md

```diff
@@ -35,37 +35,11 @@ By default, Admin Router will permit unencrypted HTTP traffic. This is not
 considered secure, and you must provide a valid TLS certificate and redirect
 all HTTP traffic to HTTPS in order to properly secure access to your cluster.
 
-Once you have a valid TLS certificate, install the certificate on each master.
+After you have a valid TLS certificate, install the certificate on each master.
 Copy the certificate and private key to a well known location, such as under
-`/etc/ssl/certs`. It's suggested that you change the permissions of the
-private key to not be world-readable. Edit the Admin Router `nginx.conf`, by
-opening the file located at
-`/opt/mesosphere/active/adminrouter/nginx/conf`. Modify the nginx
-configuration `server` section, to look like this:
-
-```
-  server {
-    listen 80 default_server;
-    server_name _;
-    return 301 https://$host$request_uri;
-  }
-
-  server {
-      listen 443 ssl spdy default_server;
-      ssl_certificate /etc/ssl/certs/your-cert.crt;
-      ssl_certificate_key /etc/ssl/certs/your-key.key;
-
-      # rest of config intentionally omitted
-```
-
-In the block above, we're adding a new `server` section above the existing one
-which will redirect all HTTP traffic to HTTPS. We've also removed the
-`listen 80 ...` directive from the second server section. After making the
-changes, restart Admin Router:
-
-```console
-# systemctl restart dcos-adminrouter
-```
```

Comment:
@joel-hamill this diff still tells me that we are about to remove the instructions for manually specifying the Open DC/OS Admin Router server certificate. Please point me to where these instructions will live instead, otherwise I cannot confidently ack this change.